### PR TITLE
Add LanguageTool icon

### LINKED ISSUE #11512: Request: LanguageTool
### Brand Name

LanguageTool 

### Website

https://languagetool.org

### Popularity Metric

- Similarweb Global Rank: [8,223](https://www.similarweb.com/website/languagetool.org)
- GitHub stars: [11.9k](https://github.com/languagetool-org/languagetool)

### Official Resources for Icon and Color

- SVG: https://languagetool.org/images/logo_color.svg
- Color: `#badfff`

### Additional Comments

Complete logo kit available: https://languagetool.org/media-assets#jump-assets

### PR DESCRIPTION
**Pull request by davidbenv**
languagetool

**Issue:** closes #11512

**Popularity metric:** 

Similarweb Global Rank: [8,223](https://www.similarweb.com/website/languagetool.org)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

### CODE REVIEW

**Comment by uncenter** [comment]
That preview URL isn't shareable - please copy/download the preview output and paste it into the PR description.

**Comment by davidbenv** [comment]
Is it nicely done now?

**Comment by uncenter** [comment]
> Is it nicely done now?

Looks good! I'll check it out in a moment.

**Comment by davidbenv** [comment]
I'd say the checks will success now.

**Comment by uncenter** [comment]
Please don't update this branch unnecessarily. Maintainers will update before merging to ensure checks still pass.

**Review by uncenter — CHANGES_REQUESTED** [review]
Icon itself looks good though!

File: _data/simple-icons.json
Comment (uncenter) [inline _data/simple-icons.json:8536]:
```suggestion
            "source": "https://languagetool.org"
```
As per https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines we should link to the page the SVG is found on (footer of the home page) instead of directly to the SVG.

**Review by uncenter — APPROVED** [review]

**Pull request merged**
